component/storage/mysql/go.mod

```diff
@@ -0,0 +1,6 @@
+// Copyright SecureKey Technologies Inc. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+module github.com/hyperledger/aries-framework-go-ext/component/mysql
```

Comment:
Are we having `component` as parent folder ? I would have expected `storage/mysql `.

Reply:
yes

Reply:
should be component/storage/mysql